cpg_workflows/large_cohort/ancestry_pca.py

```diff
@@ -57,6 +58,9 @@ def add_background(
         # combine dense dataset with background population dataset
         dense_mt = dense_mt.union_cols(background_mt)
         sample_qc_ht = sample_qc_ht.union(ht, unify=allow_missing_columns)
+
+    if drop_columns:
+        sample_qc_ht = sample_qc_ht.drop(drop_columns)
```

Comment:
I think `get_config()['large_cohort']['pca_background'].get('drop_columns')` returns the list (i.e. `['autosomal_mean_dp']`).

If this is the case then `sample_qc_ht.drop()` will need unpacking (see [here](https://hail.is/docs/0.2/hail.MatrixTable.html#hail.MatrixTable.drop))

So should become:
`sample_qc_ht = sample_qc_ht.drop(*drop_columns)`

```suggestion
        sample_qc_ht = sample_qc_ht.drop(*drop_columns)
```

Reply:
Thanks; you're right, of course. I misread that documentation as saying it would DWIM if given a list…